Gemfile.lock

```diff
@@ -0,0 +1,83 @@
+PATH
+  remote: .
+  specs:
+    just-the-docs (0.4.0.rc3)
+      jekyll (>= 3.8.5)
```

Comment:
I think `jekyll (>= 3.8.5)` is outdated – building on [GH Pages uses 3.9.2](https://pages.github.com/versions/). But then we probably also need to add `kramdown-gfm-parser` in the gemspec.

Reply:
If we lock the build to Jekyll 3 instead of 4, agreed; I can change the gemspec first (which is what would be necessary to fix this problem), and then rebuild the `Gemfile.lock`. It would have to wait for our next RC/full release.

Reply:
Also when insisting on Jekyll 4, surely the gemspec needs updating (e.g., to `~> 4.3`).